queries.sql

```diff
@@ -1,3 +1,49 @@
 /*Queries that provide answers to the questions from all projects.*/
+SELECT * from animals WHERE name Like '%mon';
+SELECT name from animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+SELECT name from animals WHERE neutered = true  AND escape_attempts < 3;
+SELECT date_of_birth from animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered = true;
+SELECT * from animals WHERE name <>'Gabumon';
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
-SELECT * from animals WHERE name = 'Luna';
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+-- Delete all records & creates a savepoint for transaction
+BEGIN;
+DELETE FROM animals; 
+ROLLBACK; 
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT delete_animals;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT delete_animals;
```

Comment:
### Optional
- [ ] It would be better if we use different `SAVEPOINT` variables for our delete and update operations. In this case, we have `delete_animals` for both SAVEPOINTs

Reply:
Alright, thank you